parsec/core/gui/device_recovery_export_widget.py

```diff
@@ -95,11 +95,16 @@ def __init__(
             )
 
     def _on_select_file_clicked(self) -> None:
-        key_dir = QDialogInProcess.getExistingDirectory(
-            self, translate("TEXT_EXPORT_KEY"), str(Path.home())
-        )
-        if key_dir:
-            self.label_file_path.setText(key_dir)
+        key_file = QDialogInProcess.getSaveFileName(
+            self,
+            translate("TEXT_EXPORT_KEY"),
+            str(Path.home() / get_recovery_device_file_name(self.get_selected_device())),
+            "*" + RECOVERY_DEVICE_FILE_SUFFIX,
```

Comment:
Use f-strings instead.

Reply:
Agreed, but it felt more readable this way.